point-of-care-poc/installation-and-configuration/roles.md

```diff
@@ -1,19 +1,19 @@
-| Personnel               | Role                       | Location    |
-|-------------------------|----------------------------|-------------|
-| Receptionist            | Reception                  | Reception   |
-| Triage Nurse            | Triage                     | Triage      |
-| Clinician               | HIV Clinic                 | HIV Clinic  |
-| Clinician               | OPD Clinic                 | OPD Clinic  |
-| Clinician               | MCH Clinic                 | MCH Clinic  |
-| Clinician               | TB Clinic                  | TB Clinic   |
-| Laboratory Technician   | Organizational: Laboratory | Laboratory  |
-| Pharmacist/Dispensery   | Organizational:Pharmacy    | Pharmacy    |
-| Data Entrant/Data Clerk | Data Clerk                 | All         |
-| Data Manager            | General Data Manager       | All Reports |
-| Data Manager            | EID Data Manager           | EID Reports |
-| Data Manager            | HTS Data Manager           | HTS Reports |
-| Data Manager            | MCH Data Manager           | MCH Reports |
-| Data Manager            | HIV Data Management        | HIV Reports |
-| Data Manager            | TB Data Manager            | TB Reports  |
-| Data Manager            | OPD Data Manager           | OPD Reports |
-| Data Manager            | SMC Data Manager           | SMC Reports |
\ No newline at end of file
+| Personnel               | Role                       | Location     | Access To   |
+|-------------------------|----------------------------|--------------|-------------|
+| Receptionist            | Reception                  | Reception    |             |
+| Triage Nurse            | Triage                     | Triage       |             |
+| Clinician               | HIV Clinic                 | HIV Clinic   |             |
+| Clinician               | OPD Clinic                 | OPD Clinic   |             |
+| Clinician               | MCH Clinic                 | MCH Clinic   |             |
+| Clinician               | TB Clinic                  | TB Clinic    |             |
+| Laboratory Technician   | Organizational: Laboratory | Laboratory   |             |
+| Pharmacist/Dispensary   | Organizational:Pharmacy    | Pharmacy     |             |
+| Data Entrant/Data Clerk | Data Clerk                 | All Locations|             |
```

Comment:
@slubwama add All Locations to each of the data manager roles for consistency

Reply:
done